test/config/config_dbs_one_writer2.json

```diff
@@ -0,0 +1,18 @@
+{
+    "base": "/dbs-one-writer",
+    "port": 8990,
+    "staticdir": "./static",
+    "dbfile": "./test/dbfile_1",
+    "lexicon_file": "./static/lexicon_writer.json",
+    "api_parameters_file": "./static/parameters.json",
+    "server_type": "DBSWriter",
+    "servercrt": "",
+    "serverkey": "",
+    "hkey": "",
+    "updateDNs": 60,
+    "log_file": "/tmp/dbs2go_writer_1_2.log",
+    "verbose": 2,
+    "limiter_rate": "10000-s",
+    "max_db_connections": 1,
```

Comment:
We can't use use it in this test since it diminish the purpose of the race testing. If we have only one db connection then all HTTP concurrent calls will be sequential.